libs/@guardian/source-foundations/src/utils/typography.ts

```diff
@@ -0,0 +1,16 @@
+import type { TypographyPreset } from '../typography/types';
+
+/*
+ * Convert `font-size` value in typography preset from rem to px
+ */
+export const presetToPx = (preset: TypographyPreset) => {
+	const REGEX_FONT_SIZE = /font-size:\s(\d+\.\d+)rem/;
+
+	const matches = preset.match(REGEX_FONT_SIZE);
+	if (matches?.[1]) {
+		const pxVal = parseFloat(matches[1]) * 16;
+		return preset.replace(REGEX_FONT_SIZE, `font-size: ${pxVal}px`);
+	}
+
+	return preset;
```

Comment:
You can also use a [replacer function as part of `String.prototype.replace`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_the_replacement), which could make this simpler:

```ts
return preset.replace(REGEX_FONT_SIZE, (_, match) => `font-size: ${parseFloat(match) * 16}px`)
```